CellProfiler64.iss

```diff
@@ -114,7 +113,20 @@ Message := 'This build can only run on a 64-bit operating system, but yours is 3
       Result := False;
       End
     else
-      Result := True;
+//
+// Check taken from the following stackoverflow post:
+//
+// http://stackoverflow.com/questions/1297773/check-java-is-present-before-installing
+//
+      RegQueryStringValue(HKLM, 'SOFTWARE\JavaSoft\Java Runtime Environment', 'CurrentVersion', JavaVer);
+      if Length( JavaVer ) > 0 then Begin
```

Comment:
OK. Version is 1.6+ at this point. There's some code at the link in the comments that we can cut and paste.